lib/plugins/Label_H1_POS.ts

```diff
@@ -17,59 +18,80 @@ export class Label_H1_POS extends DecoderPlugin {
-
-    // Style: POSN33225W079428,SCOOB,232933,340,ENEME,235712,FETAL,M42,003051,15857F6
-    let variant4Regex = /^POS(?<lat>[NS])(?<lat_coord>[0-9]+)(?<long>[EW])(?<long_coord>[0-9]+),(?<waypoint1>[a-zA-Z0-9]*),(?<unknown1>[0-9]*),(?<unknown2>[0-9]*),(?<waypoint2>[a-zA-Z0-9]*),(?<unknown3>[0-9]*),(?<waypoint3>[a-zA-Z0-9]*).(?<temp_sign>[MP])(?<temperature>[0-9]*),(?<unknown4>[0-9]*),(?<unknown5>[a-zfA-Z0-9]*)$/;
-
-    let results;
-    if (results = message.text.match(variant1Regex)) {
-
-      decodeResult = this.decodePositionRoute(decodeResult, results, options);
-
+    const checksum = message.text.slice(-4);
+    //strip POS and checksum
+    const data = message.text.substring(3, message.text.length-4);
+    const fields = data.split(',');
+    
+    if(fields.length==1 && data.startsWith('/RF')) {
+      decodeResult.raw.route = data.substring(3,data.length).split('.').map((leg: string) => {return {name: leg}});
       decodeResult.formatted.items.push({
-        type: 'aircraft_timestamp',
-        code: 'TIMESTAMP',
-        label: 'Aircraft Timestamp',
-        value: DateTimeUtils.UTCDateTimeToString(results.groups.date, results.groups.timestamp),
+        type: 'aircraft_route',
+        code: 'ROUTE',
+        label: 'Aircraft Route',
+        value: RouteUtils.routeToString(decodeResult.raw.route),
       });
 
-      decodeResult.remaining.text = `${results.groups.unknown1},${results.groups.unknown2},${results.groups.unknown3},${results.groups.unknown4},${results.groups.unknown5},${results.groups.unknown6},${results.groups.unknown7}`;
-
-      decodeResult.decoded = true;
-      decodeResult.decoder.decodeLevel = 'partial';
-
-    } else if (results = message.text.match(variant2Regex)) {
-
-      decodeResult = this.decodePositionRoute(decodeResult, results, options);
-
-      decodeResult.raw.groundspeed = Number(results.groups.groundspeed);
-
+      decodeResult.raw.checksum = Number("0x"+checksum);
       decodeResult.formatted.items.push({
-        type: 'aircraft_groundspeed',
-        code: 'GSPD',
-        label: 'Aircraft Groundspeed',
-        value: `${decodeResult.raw.groundspeed}`
-      });
-
-      decodeResult.remaining.text = `${results.groups.unknown1},${results.groups.unknown2},${results.groups.unknown3},${results.groups.unknown4},${results.groups.unknown5},${results.groups.unknown6},${results.groups.unknown7},${results.groups.unknown8},${results.groups.unknown9}`;
-
+        type: 'message_checksum',
+        code: 'CHECKSUM',
+        label: 'Message Checksum',
+        value: '0x' + ('0000' + decodeResult.raw.checksum.toString(16)).slice(-4),
+      });    
+      
       decodeResult.decoded = true;
+      // Once we know what RF stands for, I feel comfortable marking this full
```

Comment:
ChatGPT thinks it might be "Route Fix" - i haven't found other messages that start with `POS/` to know for sure

Reply:
That seems likely!